js/index.js

```diff
@@ -12,61 +12,44 @@
 
 import {Platform} from 'react-native';
 import {RNCNetInfo, NetInfoEventEmitter} from './nativeInterface';
+import type {
+  NetInfoData as _NetInfoData,
+  NetInfoType as _NetInfoType,
+  NetInfoEffectiveType as _NetInfoEffectiveType,
+} from './types';
+
+export type NetInfoData = _NetInfoData;
+export type NetInfoType = _NetInfoType;
+export type NetInfoEffectiveType = _NetInfoEffectiveType;
 
 const DEVICE_CONNECTIVITY_EVENT = 'networkStatusDidChange';
+const CHANGE_EVENT_NAME = 'connectionChange';
```

Comment:
I remember seeing some discussion that event names should maybe all be package-prefixed (e.g. `netinfo.connectionChange`) or similar since it seems there is no ability to detect event source and the event namespace is global. Not sure if there is a backward-compatible / deprecate-then-purge way to do that but since I saw the event name moving it popped into mind